### DISCUSSION TITLE
edit in WYSIWYG

### DISCUSSION THREAD

**Discussion by Jami105** [microsoft/vscode-discussions#2453] [Extension Development QnA] [answered]
I am new to VScode, so thank you for your help! 
I want to see and edit my existing files in WYSIWYG with a .php extension. 
Can anyone help me with a PAID remote lesson. 
Thank you!

PS: I installed HTML Wysiwig and PHP Intelphense but "VIEW" isn't intuitive

**Comment by alefragnani** [comment, marked as answer]
Hi @Jami105 ,

I never used the HTML Wysiwyg extension so I'm not entirely sure how powerful it is. But, since you need exactly what this extension proposes, I would suggest you to reach out the authors, on their ssues repo for questions about how to use it.

BTW, for future questions about VS Code usage, please use the VS Code repo (https://github.com/microsoft/vscode/issues) instead, because this discussions forum is for extension development only for now.

Hope this helps

**Comment by Jami105** [comment]
Thank you!
